image/tarexport/save.go

```diff
@@ -191,15 +189,13 @@ func (s *saveSession) save(outStream io.Writer) error {
 	manifestFileName := filepath.Join(tempDir, manifestFileName)
 	f, err := os.OpenFile(manifestFileName, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0644)
 	if err != nil {
-		f.Close()
 		return err
 	}
+	defer f.Close()
```

Comment:
I think this still needs to be addressed. The immediate close was removed. But the deferred close might update the modification time, which would defeat the purpose of `Chtimes`.